data/mods/gen6/random-teams.js

```diff
@@ -22,7 +22,7 @@ class RandomGen6Teams extends RandomGen7Teams {
 	 */
 	randomSet(template, teamDetails = {}, isLead = false) {
 		let baseTemplate = (template = this.dex.getTemplate(template));
-		let species = template.species;
+		let species = template.name;
 
 		if (!template.exists || !template.randomBattleMoves && !this.dex.data.Learnsets[template.id]) {
 			// GET IT? UNOWN? BECAUSE WE CAN'T TELL WHAT THE POKEMON IS
```

Comment:
In hindsight I feel like this should be Missingno, just to make it clearer that there's a bug here.

Reply:
@TheImmortal

Reply:
It's okay; I wrote this code; the joke in the comments is my own; you can change it without getting his permission.

Reply:
Let’s not change it to a Pokémon that doesn’t exist...

Reply:
There's literally a crash right below this so it's clear enough.

Reply:
Yes, the change is relatively minor and unlikely to break anything; what's your point?

Reply:
On second thought, I guess you're right. This should never happen anyway but if it does, we probably want it to be obvious and also not crash the entire server imo. I will make a change.